crates/torii/core/src/model.rs

```diff
@@ -68,11 +73,14 @@ impl ModelReader<Error> for ModelSQLReader {
     }
 
     async fn schema(&self) -> Result<Ty, Error> {
+        let model_selector =
+            get_selector_from_name(&self.name).map_err(error::ParseError::NonAsciiName)?;
```

Comment:
In the future we should directly have the selector pre-computed instead of the name. We may leave a comment to ensure the update once we will identify models by their selectors.